mmv1/products/dialogflowcx/Flow.yaml

```diff
@@ -62,9 +73,11 @@ parameters:
 properties:
   - !ruby/object:Api::Type::String
     name: 'name'
-    output: true
+    default_from_api: true
```

Comment:
FWIW I misunderstood the question regarding reading from the API so I'm not sure this design will work out - sorry for the confusion! I'll take a look over the designs & offer some thoughts since we chatted about this earlier but ultimately Sarah is the reviewer here.